pkgs/top-level/python-packages.nix

```
@@ -601,6 +601,8 @@ in {
 
   compiledb = callPackage ../development/python-modules/compiledb { };
 
+  confuse = disabledIf (!isPy3k) (callPackage ../development/python-modules/confuse { });
```

move this into expression as well
```suggestion
  confuse = callPackage ../development/python-modules/confuse { };
```